README.md

```diff
@@ -31,19 +31,27 @@
 
 ## Examples
 
-
 ```python
 from pathlib import Path
 from pyEDAA.ProjectModel import Project, Design, FileSet, VHDLSourceFile
 
-projectPath = Path("temp/project")
-project = Project("project", rootDirectory=projectPath)
-design = Design("design", project=project)
-fileset = FileSet("uart", Path("src/uart"), design=design)
+print(f"Current working directory: {Path.cwd()}")
+projectDirectory = Path.cwd() / "../project"
+print(f"Project directory: {projectDirectory!s} - {projectDirectory.exists()}")
+
+project = Project("myProject", rootDirectory=projectDirectory)
+designA = Design("designA", project=project, directory=Path("designA"))
+designAFileset = FileSet("srcA", design=designA)
+for vhdlFilePath in designAFileset.ResolvedPath.glob("*.vhdl"):
+	designAFileset.AddFile(VHDLSourceFile(vhdlFilePath))
+
+libFileset = FileSet("lib", Path("../lib"), design=designA)
+for vhdlFilePath in libFileset.ResolvedPath.glob("*.vhdl"):
+	libFileset.AddFile(VHDLSourceFile(vhdlFilePath))
```

Comment:
Codacy has a fix for the issue: [Column: 1](https://app.codacy.com/gh/edaa-org/pyEDAA.ProjectModel/pullRequest?prid=8183721)
```suggestion
 libFileset.AddFile(VHDLSourceFile(vhdlFilePath))
```